app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -386,17 +384,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
                     return;
                 }
                 break;
-            case UPGRADE_APP:
-                if (resultCode == RESULT_CANCELED) {
-                    return;
-                } else if(resultCode == RESULT_OK) {
```

Comment:
@phillipm I was actually wondering this and then forgot to ask- why do we no longer need to make the user logout after an update?